### DISCUSSION TITLE
Greetings, I am new, I need a learning path...

### DISCUSSION THREAD

**Discussion by GeoEducator** [microcks/microcks#1245] [Community]
I am new, I need a learning path...

**Comment by lbroudoux** [comment]
Hey there, welcome to Microcks community!

Did you get a try to our [Getting Started](https://microcks.io/documentation/tutorials/getting-started/)? It's the best way to check how Microcks mocks are working. Then you can pursue on the [Getting Started with Tests](https://microcks.io/documentation/tutorials/getting-started-tests/) to explore how we're doing contract/conformance testing with Microcks.

From there, you've got the option to go to some other tutorials specialized on different technologies like GraphQL, gRPC, etc.. See https://microcks.io/documentation/tutorials/
OR
to access the How-to Guides on advanced features like [applying constraints to mocks](https://microcks.io/documentation/guides/usage/mocks-constraints/), [integrating it in your development process](https://microcks.io/documentation/guides/usage/developing-testcontainers/) and so on...

You can finally access the detailed [explanations](https://microcks.io/documentation/explanations/) and [references](https://microcks.io/documentation/references/) to get in-depth view of the techno.

Let us know what you're looking or!
